src/cata_tiles.h

```diff
@@ -720,8 +744,11 @@ class cata_tiles
         static std::optional<point_bub_ms> tile_to_player(
             const point &colrow, const point_bub_ms &o,
             const point &base_tile_cnt, bool iso );
-        point player_to_tile( const point &pos ) const;
-        point player_to_screen( const point &pos ) const;
+        point player_to_tile( const point_bub_ms &pos ) const;
+        point player_to_screen( const point_bub_ms &pos ) const;
+        point player_to_screen( const point_abs_omt &pos ) {
+            return player_to_screen( point_bub_ms( pos.raw() ) );
```

Comment:
I guess this is what you referred to as "unsavory type conversion"? Is it possible to add a comment here about how this conversion from `point_abs_omt` to `point_bub_ms` is ok?

Reply:
I did add a comment by the first occurrence (line 501), but I can add it to all of them, if that's deemed to be better.
My reasoning was that it would cause a bit of clutter to repeat it, but I can easily be convinced to add to in all locations.

Reply:
Looks good, thanks. And nice work! :heart:

Reply:
Also, wanted to add, and I'm not sure this is the right venue for it, but here goes: så snyggt gjort med alla andra koordinatfixar. Allt blir så mycket tydligare tack vare dina ändringar, och ändringar jag skulle själv velat göra blir så mycket enklare tack vare dig. Thank you so much, and happy holidays. Gott nytt år du underbara person! :heart: